logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/conversation/ObserveMemberDetailsByIdsUseCaseTest.kt

```diff
@@ -118,4 +124,37 @@ class ObserveMemberDetailsByIdsUseCaseTest {
             awaitComplete()
         }
     }
+
+    @Test
+    fun givenUserIsNotKnown_whenObservingMembers_thenDoNotBlockTheFlowWaitingForItsData() = runTest {
```

Comment:
@vitorhugods do you think this one is enough to prevent it from occurring again?

Reply:
I think so, at least for this UseCase.

Something to keep in mind in the future.

I guess I and other people were a bit too optimistic on data always being there, constraints in the database, etc.

In the end, due to the focus on Privacy, users will be deleted for good, and we need to accept and handle that. It's a change of mindset: missing data is a feature, really.